app/views/users/index.html.erb

```diff
@@ -1,54 +1,42 @@
 <% provide :title, 'Members' %>
 <div class="row">
-  <div class="col-xs-12 col-sm-8">
-    <h1>All members</h1>
-    <% if @users.empty? %>
+  <div class="col-xs-12 col-sm-12">
+    <h1>Members Directory</h1>
+    <% if @users_count == 0 %>
         <p>It is a lonely planet we live in&#8230;</p>
     <% else %>
-        <p>Check out our <%= @users.count %> awesome volunteers from all over the globe!</p>
+        <p>Check out our <%= @users_count %> awesome volunteers from all over the globe!</p>
     <% end %>
   </div>
-  <div class="col-sm-4">
+  <% if @current_user %>
+    <div class="col-sm-12 well">
+      <p>Filter users by</p>
+      <%= form_tag(users_path, remote: true, method: "get", class: "form-inline filters-users-advanced") do %>
```

Comment:
Also, if the request isn’t done with ajax, you get the added benefit of keeping the URL stateful so refreshing the page won’t lose your results. You also get rid of the extra `.js` format option, so the controller will look a lot more cleaner, but I am curious to hear why you guys chose this route to begin with.